source/core/parameters.cc

```diff
@@ -186,6 +186,51 @@ namespace Parameters
     prm.leave_subsection();
   }
 
+  void
+  NonNewtonian::declare_parameters(ParameterHandler &prm)
+  {
+    prm.enter_subsection("non newtonian");
+    {
+      prm.declare_entry("model",
+                        "carreau",
+                        Patterns::Selection("carreau"),
+                        "Non newtonian model "
+                        "Choices are <carreau>.");
+      prm.declare_entry("viscosity_0",
+                        "1",
+                        Patterns::Double(),
+                        "Viscosity at rest");
+      prm.declare_entry("viscosity_inf",
+                        "2",
+                        Patterns::Double(),
+                        "Viscosity for an infinite constraint");
+      prm.declare_entry("lambda", "1", Patterns::Double(), "Relaxation time");
+      prm.declare_entry("a", "1", Patterns::Double(), "Relaxation time");
```

Comment:
Are both of this relaxation time?

Reply:
No, thank you for the comment.

Reply:
For the name of the example, I used Carreau since in is the name of a person, but I can totally switch to carreau.